Tools/peg_generator/.ruff.toml

```diff
@@ -0,0 +1,35 @@
+extend = "../../.ruff.toml"  # Inherit the project-wide settings
+extend-exclude = [
+    # Generated files:
+    "Tools/peg_generator/pegen/grammar_parser.py",
+]
+
+[lint]
+select = [
+    "F",  # Enable all pyflakes rules
+    "I",  # Enable all isort rules
+    "UP",  # Enable all pyupgrade rules by default
+    "RUF100",  # Ban unused `# noqa` comments
+    "PGH004",  # Ban blanket `# noqa` comments (only ignore specific error codes)
+]
+ignore = [
+    # Unnecessary parentheses to functools.lru_cache: just leads to unnecessary churn.
+    # https://github.com/python/cpython/pull/104684#discussion_r1199653347.
+    "UP011",
+    # Use format specifiers instead of %-style formatting.
+    # Doesn't always make code more readable.
+    "UP031",
+    # Use f-strings instead of format specifiers.
+    # Doesn't always make code more readable.
```

Comment:
Fine to ignore these.

If you like, you could temporarily enable the locally, do a test run and see if it does make any of them more readable, and just commit those changes?

Reply:
I don't think we have a lot of cases where not using f-strings is worse, but checking won't hurt indeed 👍 

I am happy if most of the changes are an improvement in any case :)

Reply:
Yeah, there were two cases, where `f` string is better :)